src/EA.Weee.Web/Areas/Aatf/Mappings/ToViewModel/AatfDataToHomeViewModelMap.cs

```diff
@@ -53,5 +55,32 @@ public HomeViewModel Map(AatfDataToHomeViewModelMapTransfer source)
 
             return Model;
         }
+
+        private List<AatfData> RemoveOlderAatfs(List<AatfData> source)
+        {
```

Comment:
Can do this with a linq query by grouping by AATFid and taking the top 1 of each, be tidier